src/pages/signin/SignInPage.js

```diff
@@ -103,8 +113,8 @@ class SignInPage extends Component {
 SignInPage.propTypes = propTypes;
 SignInPage.defaultProps = defaultProps;
 
-export default withOnyx({
+export default withLocalize(withOnyx({
     account: {key: ONYXKEYS.ACCOUNT},
     credentials: {key: ONYXKEYS.CREDENTIALS},
     session: {key: ONYXKEYS.SESSION},
-})(SignInPage);
+})(SignInPage));
```

Comment:
Let's use `compose` for when we wanna combine multiple HOCs together. Something like this:

```js
export default compose(
    withLocalize,
    withOnyx({
        account: {key: ONYXKEYS.ACCOUNT},
        credentials: {key: ONYXKEYS.CREDENTIALS},
        session: {key: ONYXKEYS.SESSION},
    }),
)(SignInPage);
```

Reply:
Done. Thanks for the tip.